systemuser.sh

```diff
@@ -84,12 +86,25 @@ sed -i "s/IRkernel::main()/options(bitmapType='cairo');IRkernel::main()/g" $KERN
 chown -R $USER:$USER $JPY_DIR $JPY_LOCAL_DIR $IPYTHONDIR
 export SWAN_ENV_FILE=/tmp/swan.sh
 
-sudo -E -u $USER sh -c 'source $LCG_VIEW/setup.sh \
+sudo -E -u $USER sh -c 'mkdir -p $SWAN_HOME/SWAN_projects/ \
+                        && source $LCG_VIEW/setup.sh \
```

Comment:
If the source fails (as it happens sometimes), then the mkdir won't execute. If it's the first time the users is using SWAN and doesn't have the projects folder, the system will crash because it's going to try to open it.

Reply:
I see, understood